Schlange.py

```diff
@@ -84,42 +84,42 @@ class KeywordTranslator:
     'KeyboardInterrupt': 'Tastaturunterbrechung',
     }
     errors_with_placeholders = {
-    'unsupported operand type': 'nicht unterstützter Operandentyp',
-    'division by zero': 'Division durch Null',
-    'name \'{}\' is not defined': 'Name \'{}\' ist nicht definiert',
-    'list index out of range': 'Listenindex außerhalb des gültigen Bereichs',

```

Comment:
And there is no real reason for non-first words to start with an upper-case letter, and the english version has no upper-case